confusables/__init__.py

```diff
@@ -42,17 +43,19 @@ def confusable_characters(char):
         return [char]
     return None
 
+
 def confusable_regex(string, include_character_padding=False):
-    space_regex = "[\*_~|`\-\.]*" if include_character_padding else ''
+    space_regex = r"[\*_~|`\-\.]*" if include_character_padding else ''
```

Comment:
This isn't needed, and it appears to be outside of the scope of the PR

Reply:
Fixed 👍🏽